frontend/src/components/NamesList.js

```diff
@@ -0,0 +1,20 @@
+import React, { useContext } from "react";
+import { GlobalContext } from "../context/GlobalContext";
+
+function NamesList() {
+  const { nameList, deleteName } = useContext(GlobalContext);
+
+  return (
+    <div>
+      List:
+      {nameList.map((name) => (
+        <div>
+          <div key={name}>{name}</div>
```

Comment:
You were right to include that `key={name}`, because each child in a list should include a unique key, but you put it a level too low!
```suggestion
        <div key={name}>
          <div>{name}</div>
```